mne/report.py

```diff
@@ -887,11 +898,18 @@ def add_images_to_section(self, fnames, captions, scale=None,
             img_klass = self._sectionvars[section]
 
             # Convert image to binary string.
-            im = Image.open(fname)
             output = BytesIO()
-            im.save(output, format='png')
+            im = Image.open(fname)
+            if image_format == 'png':
+                im.save(output, format='png')
+            elif image_format == 'gif':
+                with open(fname, 'rb') as f:
+                    output.write(f.read())
```

Comment:
I think it was to get the binary string but if there's a better way to do it (looks like it), we should probably switch to that.